jest.config.js

```diff
@@ -40,19 +40,11 @@ module.exports = {
     "^highlight.js$": `<rootDir>/node_modules/highlight.js/lib/index.js`,
   },
   snapshotSerializers: [`jest-serializer-path`],
-  collectCoverage: useCoverage,
-  coverageReporters: [`json-summary`, `text`, `html`, `cobertura`],
-  coverageThreshold: {
-    global: {
-      lines: 45,
-      statements: 44,
-      functions: 42,
-      branches: 43,
-    },
-  },
```

Comment:
we weren't using them anyway and those were just failing because they tried to analyze fixtures - might be good idea to set them up properly with fixture ignore, but that's not in scope of this PR